playbooks/appsemblerPlaybooks/update_custom_requirements.yml

```diff
@@ -0,0 +1,104 @@
+- hosts: "edxapp-server"
+  become: true
+
+  # Update extra requirements, private requirements, 
+  # collect static assets, and restart edx app and workers
+
+  tasks:
+    - include_vars: roles/common_vars/defaults/main.yml
+    - include_vars: roles/edxapp/defaults/main.yml
+
+
+    - name: install read-only ssh key
+      copy: >
+        content="{{ EDXAPP_GIT_IDENTITY }}" dest={{ edxapp_git_identity }}
+        force=yes owner={{ edxapp_user }} mode=0600
+      when: EDXAPP_USE_GIT_IDENTITY
+
+    - name: set git fetch.prune to ignore deleted remote refs
+      shell: git config --global fetch.prune true
+      sudo_user: "{{ edxapp_user }}"
+
+    - name: install python private requirements
+      shell: >
+        {{ edxapp_venv_dir }}/bin/pip install {{ COMMON_PIP_VERBOSITY }} -i {{ COMMON_PYPI_MIRROR_URL }} --exists-action w -r {{ item }}
+        chdir={{ edxapp_code_dir }}
```

Comment:
@johnbaldwin I actually just copied this over from `roles/edxapp/tasks/deploy.yml` but omitted some comments there: 

```
  # Need to use shell rather than pip so that we can maintain the context of our current working directory; some
  # requirements are pathed relative to the edx-platform repo. Using the pip from inside the virtual environment implicitly
  # installs everything into that virtual environment.
```

I'm just taking the author's word for it :)